contrib/grinder/README.md

```diff
@@ -1,99 +1,5 @@
 # Blueflood Grinder Integration
-##Intro
-Grinder is a distributed load testing tool described [here](http://grinder.sourceforge.net/g3/getting-started.html)
 
-This code defines implementations of grinder worker threads meant to repeatedly invoke the required number of BF api calls during each "reporting interval"
+### DEPRECATED
```

Comment:
I dont think we have to do this. If its possible, mention about the new grinder script in wiki or some place appropriate.